webtau-groovy/src/test/resources/webtau-nested.cfg

```diff
@@ -0,0 +1,31 @@
+url = "http://localhost:8180"
+configValue = 'testUser'
+
+additionalUrls {
+    appOne = "http://app-one"
+    appTwo = "http://app-one"
+}
+
+parent.child = 'pc-1'
+parent.another = 'pc-2'
+
+environments {
+    dev {
+        additionalUrls {
+            appOne = "http://app-one-dev"
+        }
+    }
+
+    prod {
+        additionalUrls.appOne = "http://app-one-prod"
+        additionalUrls {
```

Comment:
Just to confirm.  If there was a `additionalUrls.appThree` which was not overriden here, it would preserve the default value?  In other words, for nested config, you can override just individual fields, don't have to override the entire object?

Reply:
@tsiq-karold good point! I am going to add a quick test.

Reply:
Nice, thanks!